### PR TITLE
Bump e2e-prow to v20170808-0a00d3e7 (using generate_tests and manual)

### PR DESCRIPTION
**Pull request by zmerlynn**
Along the way: Bump the docker tag used in `scenarios/kubernetes_e2e.py` as well.

This is primarily to pick up #3967 for #3983 

Canaries have been running since 9:45-ish with this code.

### CODE REVIEW

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -33,6 +33,9 @@
 
 ORIG_CWD = os.getcwd()  # Checkout changes cwd
 
+# Note: This variable is managed by experiment/bump_e2e_image.sh.
+DEFAULT_KUBEKINS_TAG = 'v20170808-0a00d3e7'
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: seems the image has not been pushed yet?

Comment (zmerlynn) [inline scenarios/kubernetes_e2e.py]: Oh, woops. Re-doing, this is changing the wrong tag.

**Comment by zmerlynn** [comment]
PTAL - fixed `bump_e2e_image` to change to the right tag. :)

**Comment by zmerlynn** [comment]
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-canary/650/ & https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-prow-canary/17176/ are relevant jobs, but again, `:latest` was bumped a bit ago (and when I pushed again the layers weren't changed).

**Pull request merged**